### DISCUSSION TITLE
Explorer Patcher Effecting Registry Key Export

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46197] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Since the new Windows 11 cumulative updates over the past month I am no longer able to Export Registry Keys. I found a workaround that is simply though. Uninstall EP 22621.608.51.6 using Programs and Features. Once I export the Key I reinstall EP. Easy enough but still I wanted to discuss this here at Github. I am just a user, not a coder.

**Comment by jge162** [comment]
Thank you for your post.

**Reply by ghost** [reply]
Thanks for that info, same thing happened to me. I have a new glitch that I cannot figure out how to repair. The chevrons for the extra QLA shortcuts have a huge white space on the right of them. (see image). I'm good with editing the Reg. Anyone got a suggestion?
![pop-up QLA](https://user-images.githubusercontent.com/34069523/219298717-88d29b2e-1f88-4239-90a7-a762f416e288.jpg)

**Comment by ghost** [comment, marked as answer]
Per my post above. I figured out what the problem was. The long labels of some shortcuts in the QLA needed to be shortened. After doing so I now have a regular sized pop-up of the extra shortcuts when I click the chevrons >>.
![removed wide white area](https://user-images.githubusercontent.com/34069523/219845025-2aece33c-f924-4f2f-b3ea-31286c798dda.jpg)
